tests/README.md

```diff
@@ -4,26 +4,35 @@ Tests are defined in the `tests` folder in this project.
 #### Running tests using Docker
 All tests can be run inside a Docker container which includes all the necessary
 Glue/Spark dependencies and simulates the environment which the Glue jobs
-will be run in. A Dockerfile is included in the `tests` directory
+will be run in. Dockerfiles for AWS versions 3.0 and 4.0 are included in the `tests` directory.
 
 To run tests locally, first configure your AWS credentials, then launch and attach
-to the docker container (see following commands)
+to the relevant docker container (see following commands)
 
-Run the following commands to run tests for the s3_to_json script (in develop).
+These scripts needs to be run inside a Docker container:
+
+- Under AWS 3.0 (Dockerfile.aws_glue_3)
+  - test_compare_parquet_datasets.py
+  - test_s3_to_json.py
+
+- Under AWS 4.0 (Dockerfile.aws_glue_4)
+  - test_json_to_parquet.py (Note that these tests deploys test resources to aws and will take several min to run)
```

Comment:
Will there be a time when we can have one docker container?

Reply:
Yes, when AWS python shell jobs support Python 3.10